cmake/autogenerated_versions.txt

```diff
@@ -2,11 +2,11 @@
 
 # NOTE: has nothing common with DBMS_TCP_PROTOCOL_VERSION,
 # only DBMS_TCP_PROTOCOL_VERSION should be incremented on protocol changes.
-SET(VERSION_REVISION 22)
```

Comment:
The `VERSION_REVISION` is dependent by our version management.

Reply:
don't modify it manually

Reply:
thanks. I reverted change for this line